t/01-test_needle.t

```diff
@@ -441,4 +441,16 @@ subtest 'workaround property' => sub {
     is($no_workaround_needle->get_property_value("glossy"),             undef, "glossy property is a string, has no value");
 };
 
+subtest 'clarify error message when needles directory does not exist' => sub {
+    $bmwqemu::vars{CASEDIR}     = '/tmp/foo';
+    $bmwqemu::vars{PRODUCTDIR}  = '/tmp/boo/products/boo';
+    $bmwqemu::vars{NEEDLES_DIR} = undef;
+    eval { needle::init };
+    like($@, qr/Can't init needles from \/tmp\/boo\/products\/boo\/needles at.*/, 'do not combine CASEDIR when the default needles directory is an absolute path');
```

Comment:
You can use https://metacpan.org/pod/Test::Exception here which is already imported

Reply:
Thanks. fixed it.